subsys/bluetooth/controller/Kconfig.df

```diff
@@ -3,15 +3,23 @@
 # Copyright (c) 2020 Nordic Semiconductor ASA
 # SPDX-License-Identifier: Apache-2.0
 
+# BT_CTLR_DF_SUPPORT is a wrapper for all DF features.
+# If only certain DF features are supported, those should be selected
+# individually.
 config BT_CTLR_DF_SUPPORT
 	bool
-	depends on !BT_CTLR_TIFS_HW
 	select BT_CTLR_DF_CTE_TX_SUPPORT
 	select BT_CTLR_DF_ANT_SWITCH_2US_SUPPORT
 	select BT_CTLR_DF_ANT_SWITCH_1US_SUPPORT
 	select BT_CTLR_DF_CTE_RX_SUPPORT
 	select BT_CTLR_DF_CTE_RX_SAMPLE_1US_SUPPORT
 
+if BT_LL_SW_SPLIT
+config BT_CTLR_DF_SUPPORT
+	bool
+	depends on !BT_CTLR_TIFS_HW
```

Comment:
`BT_CTLR_DF_SUPPORT` is used as a supported features set for Zephyr LL only. It is just a convenience wrapper. I suggest to move select part: https://github.com/zephyrproject-rtos/zephyr/pull/45022/files#diff-e9984b2a061624f67ff5705bc3295a77b97df2b1c5b4452dc706a3177bd710c1R11-R15 here and remove first definition.